.gitignore

```diff
@@ -4,6 +4,7 @@ dist
 .env
 .pnpm-debug.log
 .vscode
```

Comment:
Using JetBrains Fleet?

Reply:
Yeah have been giving it another spin; I try to look at it every couple months to see when it is in a more usable state